src/backend/mi2/mi2.ts

```diff
@@ -5,9 +5,7 @@ import { parseMI, MINode } from '../mi_parse';
 import * as linuxTerm from '../linux/console';
 import * as net from "net";
 import * as fs from "fs";
-import { posix } from "path";
-import * as nativePath from "path";
-const path = posix;
+import * as path from "path";
```

Comment:
why did you change the definition of path? It's used for SSH path building to always use the posix path.

Reply:
That was actually a bad hack. Using the explicit `path.posix` in every code below is definitely better.

Reply:
Hi @WebFreak001, @GitMensch and I had a discussion on 43cc39e7773f1122ab61cc863256d2264a865e3c where he ran into an issue when the SSH connection was to a Windows machine that the paths were expected to be in Win32 path formatting.  This change was an attempt to "detect" the path type based on the CWD and use the corresponding path manipulation APIs based on that detection.